saleor/tests/fixtures.py

```diff
@@ -1332,6 +1336,12 @@ def order_events(order):
 @pytest.fixture
 def fulfilled_order(order_with_lines):
     order = order_with_lines
+    invoice = order.invoices.create(
```

Comment:
Shouldn't we have separate fixture for this? I don't think that all tests which use `fulfilled_order` requires also invoice object.  From the tests perfomence perspective it would be better to have to fixutres.